.travis.yml

```diff
@@ -8,16 +8,10 @@ bundler_args: --without development --jobs=3 --retry=3
 sudo: false
 
 rvm:
-  - 1.8.7
-  - 1.9.3
-  - 2.0.0
-  - 2.1.10
-  - 2.2.10
-  - 2.3.8
-  - 2.4.5
-  - 2.5.3
-  - 2.6.1
-  - jruby-9.2.6.0
+  - 2.4.6
+  - 2.5.5
+  - 2.6.3
+  - jruby-9.2.7.0
```

Comment:
Update the rvm list to remove unsupported ruby versions and match the list in `simplecov` gem https://github.com/colszowka/simplecov/blob/master/.travis.yml